lib/codegen/generator.dart

```diff
@@ -292,7 +292,7 @@ class _DeclarationCode extends ConstExpression {
 
   List<String> get librariesUsed => []
```

Comment:
The analyzer isn't complaining, but I'll make that change since I think you're right.